### DISCUSSION TITLE
How to search ext3

### DISCUSSION THREAD

**Discussion by JobarPara08** [PabloLec/RecoverPy#95] [Q&A] [answered]
Hello and thank you for RecoverPY. But I'm using it for the first time and trying to recover a script I wrote that was stored on an external ext3 hard disk. My search don't find any result so maybe it is impossible to recover it but I saved it with VS Code maybe 30 minutes before it got deleted so do you have an idea ?

**Comment by PabloLec** [comment, marked as answer]
Hi, you should be able to search ext3 like any other file system.
You may try to search a common string or a single char to make sure your partition is indeed able to be searched. Then maybe search for another string that was in your file.
I see that you used VS Code to edit your file, which should make temporary backups of your file. With a bit of luck, try searching inside your home partition (or wherever your VS Code directory might be).

**Reply by JobarPara08** [reply]
Thx!! It worked on my home partition I was able to find my code :smiley:
